### PR TITLE
Refactor `HorizontalCard` component to ChakraUI.

### PR DESCRIPTION
**Pull request by ameeetgaikwad**
## Description
Migrate `HorizontalCard` component to Chakra.

Includes refactor of `p` and `div` of the component.

### CODE REVIEW

**Comment by minimalsm** [comment]
Hey @ameeetgaikwad, thanks for the PR 😎 

@pettinarip will be reviewing our Chakra PRs soon. Could you remove the `yarn.lock/package.json` changes from this PR please?

File: src/components/HorizontalCard.tsx [hunk 1]
```diff
@@ -33,20 +13,25 @@ export interface IProps {
...
   emoji,
-  title,
-  description,
   children,
   className,
   emojiSize,
 }) => (
-  <StyledCard className={className}>
+  <Box
+    display="flex"
```
Comment (pettinarip) [inline src/components/HorizontalCard.tsx:21]:
```suggestion
  <Flex
```

File: src/components/HorizontalCard.tsx [hunk 2]
```diff
@@ -33,20 +13,25 @@ export interface IProps {
...
+    flexGrow="0"
+    flexShrink="1"
+    flexBasis="75%"
```
Comment (pettinarip) [inline src/components/HorizontalCard.tsx:24]: These values are part of the `Content` div, not the `StyledCard`.

File: src/components/HorizontalCard.tsx [hunk 3]
```diff
@@ -33,20 +13,25 @@ export interface IProps {
...
+    borderRadius="base"
+    className={className}
+  >
     <Emoji size={emojiSize} text={emoji} />
-    <Content>
-      <Title>{title}</Title>
-      <Description>{description}</Description>
+    <Box display="flex" ml="8">
+      <Text fontSize="lg" />
+      <Box as="p" p="0.8" mt="-4" mb="2" />
```
Comment (pettinarip) [inline src/components/HorizontalCard.tsx:31]: Lets use `Text` here.

Comment (pettinarip) [inline src/components/HorizontalCard.tsx:31]:
```suggestion
      <Box as="p" opacity="0.8" mt="-4" mb="2" />
```

File: src/components/HorizontalCard.tsx [hunk 4]
```diff
@@ -1,26 +1,6 @@
 import React, { ReactNode } from "react"
-import styled from "@emotion/styled"
 import Emoji from "./OldEmoji"
```
Comment (pettinarip) [inline src/components/HorizontalCard.tsx:2]: We need to change this as well. We have a new `Emoji` component that is using Chakra.

Comment (pettinarip) [reply src/components/HorizontalCard.tsx:2]: With that change, we will have to remove the support of `emojiSize` here. It should be renamed to `emojiFontSize` and we would need to update those props on the places where we are using it.

For example, in `pages/stablecoins.tsx`
```tsx
// before
<TokenCard
  key={idx}
  emoji={token.emoji}
  description={token.description}
  emojiSize={3}
/>

// now
<TokenCard
  key={idx}
  emoji={token.emoji}
  description={token.description}
  emojiFontSize="5xl"
/>
```

**Review by pettinarip — CHANGES_REQUESTED** [review]
Hey @ameeetgaikwad! thanks for the PR! 💪🏼 

- I see that you have created a `package-lock.json` & changed the `yarn.lock`. Lets not include that here since it is not in the scope of this PR.
- I've left a few change requests about the Chakra implementation.

**Comment by pettinarip** [comment]
Hey @ameeetgaikwad I see that you have merged back the files. Do you need help with this? if it is easier for you, you can create a new PR with a clean branch. Feel free to ping me on Discord if you need help with git or anything else.

**Comment by ameeetgaikwad** [comment]
> Hey @ameeetgaikwad I see that you have merged back the files. Do you need help with this? if it is easier for you, you can create a new PR with a clean branch. Feel free to ping me on Discord if you need help with git or anything else.

yeah i'll work on a new branch

File: src/components/HorizontalCard.tsx (hunk 3 above)
Comment (ameeetgaikwad) [reply src/components/HorizontalCard.tsx:31]: Use `Text` instead of `Box`?

File: src/components/HorizontalCard.tsx (hunk 4 above)
Comment (ameeetgaikwad) [reply src/components/HorizontalCard.tsx:2]: Can you eleborate? What should I import?

File: src/components/HorizontalCard.tsx (hunk 3 above)
Comment (pettinarip) [reply src/components/HorizontalCard.tsx:31]: Yes, here you are using a Box to print a `p`. We can just use `Text` (from Chakra https://chakra-ui.com/docs/components/text/usage) which already print a `p` tag for you.

File: src/components/HorizontalCard.tsx (hunk 4 above)
Comment (pettinarip) [reply src/components/HorizontalCard.tsx:2]: You can check that here in the Dependencies section https://github.com/ethereum/ethereum-org-website/blob/dev/docs/chakra-migration-guide.md#update-dependencies

Basically you need to replace `OldEmoji` with the new emoji component located here: `src/components/Emoji`.

Comment (ameeetgaikwad) [reply src/components/HorizontalCard.tsx:2]: Should I refactro `pages/stablecoins.tsx` also? Should I ask to get assigned for it in the issues section?

**Comment by ameeetgaikwad** [comment]
> > Hey @ameeetgaikwad I see that you have merged back the files. Do you need help with this? if it is easier for you, you can create a new PR with a clean branch. Feel free to ping me on Discord if you need help with git or anything else.
> 
> yeah i'll work on a new branch

@pettinarip check this out #7893

**Pull request closed without merging**